crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
+    """
+    Read the names in ../names
+    :return: names_list
+    """
+    names_list = []
+    for name_file in glob.glob("./names/*"):
+        with open(name_file, 'r') as f:
+            first_line = f.readline()
+            first_name = first_line.split()[0]
```

Comment:
I recommend using HumanName()  nameparser to handle the name parsing....someone else handled all the fancy things in human names (like van der Waals and so on) and wrote tests to make them work.  If your name doesn't work in HumanName you can make a PR into that project and then your name will work in everyone's code if they are using nameparser.....  It would look like
```
name = HumanName(first_line)
```
then later you can get `name.first`, `name.middle`, `name.last`, `name.suffix`